doc/TUTORIAL.md

```diff
@@ -165,8 +166,7 @@ module.exports = class GemVersion extends BaseJsonService {     // (5)
   static get url() {                                            // (6)
     return {
       base: 'gem/v',
-      format: '(.+)',
-      capture: ['gem'],
+      format: ':gem',
```

Comment:
Can we also apply this to the actual gem service so that the example in the docs reflects real life

Reply:
Haha, sure.